Formula/go@1.9.rb

```diff
@@ -0,0 +1,95 @@
+class GoAT19 < Formula
+  desc "Go programming environment (1.9)"
+  homepage "https://golang.org"
+
+  stable do
+    url "https://dl.google.com/go/go1.9.4.src.tar.gz"
+    mirror "https://fossies.org/linux/misc/go1.9.4.src.tar.gz"
+    sha256 "0573a8df33168977185aa44173305e5a0450f55213600e94541604b75d46dc06"
+
+    go_version = version.to_s.split(".")[0..1].join(".")
+    resource "gotools" do
+      url "https://go.googlesource.com/tools.git",
+          :branch => "release-branch.go#{go_version}"
+    end
+  end
+
+  keg_only :versioned_formula
+
+  option "without-cgo", "Build without cgo (also disables race detector)"
+  option "without-race", "Build without race detector"
+
+  depends_on :macos => :mountain_lion
+
+  # Don't update this unless this version cannot bootstrap the new version.
+  resource "gobootstrap" do
+    url "https://storage.googleapis.com/golang/go1.7.darwin-amd64.tar.gz"
+    version "1.7"
+    sha256 "51d905e0b43b3d0ed41aaf23e19001ab4bc3f96c3ca134b48f7892485fc52961"
+  end
+
+  def install
+    (buildpath/"gobootstrap").install resource("gobootstrap")
+    ENV["GOROOT_BOOTSTRAP"] = buildpath/"gobootstrap"
+
+    cd "src" do
+      ENV["GOROOT_FINAL"] = libexec
+      ENV["GOOS"]         = "darwin"
+      ENV["CGO_ENABLED"]  = "0" if build.without?("cgo")
+      system "./make.bash", "--no-clean"
+    end
+
+    (buildpath/"pkg/obj").rmtree
+    rm_rf "gobootstrap" # Bootstrap not required beyond compile.
+    libexec.install Dir["*"]
+    bin.install_symlink Dir[libexec/"bin/go*"]
+
+    # Race detector only supported on amd64 platforms.
+    # https://golang.org/doc/articles/race_detector.html
+    if build.with?("cgo") && build.with?("race") && MacOS.prefer_64_bit?
```

Comment:
Supported platforms are all 64-bit, so remove the `if` and run the next line unconditionally.